modules/git/bootstrap.Makefile

```diff
@@ -27,3 +28,4 @@ else
 endif
 
 endif
+endif
```

Comment:
is it correct that if `d` flag is not set, we skip the entire file?
do we want to disable just the warning?

```
ifeq ($(wildcard .git),)
  ifneq (,$(findstring d,$(MAKEFLAGS)))
      $(warning disabling git bootstrapping)
  endif
else
.....
endif
```

Reply:
@aknysh good catch! That does look like the case. @vadim-hleif We only want to disable the warnings, not the logic =)